tests/fuzz/execution_context.cpp

```diff
@@ -24,38 +24,52 @@
 #include "platform.h"
 #include "test_helper.hpp"
 
+#define ONE_MB_IN_BYTE (1024 * 1024)
+
 std::vector<ebpf_handle_t>
 get_handles()
 {
+    std::vector<ebpf_handle_t> handles;
     ebpf_result_t result;
     const char* error_message = nullptr;
     bpf_object* object = nullptr;
     bpf_link* link;
     fd_t program_fd;
+    std::vector<std::string> map_names{
+        "HASH",
+        "PERCPU_HASH",
+        "ARRAY",
+        "PERCPU_ARRAY",
+        "LRU_HASH",
+        "LRU_PERCPU_HASH",
+        "QUEUE",
+        "STACK",
```

Comment:
Is there some logical order to this list?  It doesn't seem to be in alphabetic or numerical order.

Reply:
It's the same order they occur in map.o ELF file.

Reply:
Ok.  it's unfortunate that map.o uses a random ordering.  Does the order here need to match the order in the map.o file?  If so, at least add a // comment explaining that.